address-book-with-rails/server/Gemfile

```diff
@@ -32,10 +32,8 @@ end
 
 group :development do
   gem 'listen', '~> 3.2'
-  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
-  gem 'spring'
-  gem 'spring-watcher-listen', '~> 2.0.0'
```

Comment:
![image](https://github.com/user-attachments/assets/f90f7a8a-192f-4285-97ed-1bcf60fbb05a)

thread discussing spring as being the issue:
https://stackoverflow.com/a/70917991